torch/onnx/symbolic_helper.py

```diff
@@ -477,18 +477,40 @@ def _interpolate_warning(interpolate_mode):
                   "We recommend using opset 11 and above for models using this operator.")
 
 def _unsqueeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Unsqueeze", input, axes)
-    else:
-        return g.op("Unsqueeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Unsqueeze", input, axes)
+        else:
+            return g.op("Unsqueeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            return g.op("Unsqueeze", input, axes_t)
+        else:
+            raise ValueError("Opset version must be >= 13 for Unsqueeze with dynamic axes.")
 
 def _squeeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Squeeze", input, axes)
-    else:
-        return g.op("Squeeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
```

Comment:
basically all the same comments apply to this function as applied to the _unsqueeze_helper above:
1. Use implicit `else` to decrease indentation
2. Handle errors first.
3. Add source range to error messages.

Reply:
Pretty helpful comments, thanks!